nginx/assets/monitors/5xx.json

```diff
@@ -0,0 +1,32 @@
+{
+	"name": "[NGINX] 5xx Errors",
```

Comment:
Can we change the title to something more descriptive like "5xx Errors higher than normal"?

Reply:
Updated